### PR TITLE
Remove 'tense' from the reference to 'Classes' section

### PR DESCRIPTION
**Pull request by navneetkarnani**
The current text refers to the classes section in the 'past tense' when in the handbook flow it is in the future. I am proposing that the sentence be changed to 'tense neutral' to account for the reading sequence changes, if any, that happen, or if the reader follows a different flow.

### CODE REVIEW

File: packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ console.log(stringNumeric.add(stringNumeric.zeroValue, "test"));
 
 Just as with interface, putting the type parameter on the class itself lets us make sure all of the properties of the class are working with the same type.
 
-As we covered in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
+As cover in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
```
Comment (orta) [inline packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md]:
```suggestion
As we will cover in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
```

There isn't really a tense neutral way to say this in English ("see also" maybe) but I think switching the tense is reasonable

Comment (navneetkarnani) [reply packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md]: I missed the "we" between the "As" and "cover". See if that has a different reading impact. Just pushed a change to the branch for that.

Comment (orta) [reply packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md]: I agree, I like your version 👍🏻

**Comment by orta** [comment]
Thanks

**Pull request merged**